udev-trigger-generator

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+# Copyright (c) 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# bsc#1174287: On PowerPC, exclude memory devices from udev coldplug
+export PATH=/usr/bin
```

Comment:
I don't think that's needed, PID1 is supposed to start generators with a clean env.

Reply:
well if that's not needed, there's no point to keep it as it's just confusing otherwise.

Reply:
I disagree. This way the code can be read without prior knowledge about the environment which PID 1 may or may not have set up. The man page **systemd.generator(7)** doesn't say anything about the environment in which the generators are run.